tests/test_dtel.py

```diff
@@ -211,7 +211,112 @@ def test_DtelQueueReportAttribs(self, dvs, testlog):
                     assert False
 
         tbl._del("Ethernet0|0")
-    
+
+    def test_DtelFlowWatchlist(self, dvs, testlog):
+        self.db = swsscommon.DBConnector(4, dvs.redis_sock, 0)
+        self.adb = swsscommon.DBConnector(1, dvs.redis_sock, 0)
+        self.table = "DTEL_FLOW_WATCHLIST"
+
+        fieled_1=[("PRIORITY", "30"),
+                 ("ETHER_TYPE", "0x800"),
+                 ("L4_DST_PORT", "1674"),
+                 ("FLOW_OP", "POSTCARD"),
+                 ("REPORT_ALL_PACKETS", "FALSE"),
+                 ("DROP_REPORT_ENABLE", "TRUE"),
+                 ("TAIL_DROP_REPORT_ENABLE", "TRUE")]
+        fieled_2=[("PRIORITY", "40"),
+                 ("ETHER_TYPE", "0x800"),
+                 ("L4_DST_PORT", "1674"),
+                 ("FLOW_OP", "POSTCARD"),
+                 ("REPORT_ALL_PACKETS", "TRUE"),
+                 ("DROP_REPORT_ENABLE", "FALSE"),
+                 ("TAIL_DROP_REPORT_ENABLE", "FALSE")]
+        fieled_3=[("PRIORITY", "50"),
+                 ("ETHER_TYPE", "0x800"),
+                 ("L4_DST_PORT", "1674"),
+                 ("FLOW_OP", "POSTCARD"),
+                 ("REPORT_ALL_PACKETS", "TRUE")]
+        fieled_4=[("PRIORITY", "60"),
+                 ("ETHER_TYPE", "0x800"),
+                 ("L4_DST_PORT", "1674"),
+                 ("REPORT_ALL_PACKETS", "TRUE"),
+                 ("DROP_REPORT_ENABLE", "TRUE"),
+                 ("TAIL_DROP_REPORT_ENABLE", "TRUE")]
+        fieled_5=[("PRIORITY", "70"),
+                 ("ETHER_TYPE", "0x800"),
+                 ("L4_DST_PORT", "1674"),
+                 ("FLOW_OP", "NOP"),
+                 ("REPORT_ALL_PACKETS", "FALSE"),
+                 ("DROP_REPORT_ENABLE", "TRUE"),
+                 ("TAIL_DROP_REPORT_ENABLE", "TRUE")]
+        listfield = [fieled_1, fieled_2, fieled_3, fieled_4, fieled_5]
+
+        for field in listfield:
+            k = listfield.index(field)
+            rule = "RULE-" + str(k)
+            self._create_dtel_acl_rule(self.table, rule, field)
+            time.sleep(1)
```

Comment:
There are a lot of time.sleep(1) in this code, probably more than are needed.

Reply:
done